fv3core/stencils/remapping_part1.py

```diff
@@ -303,19 +303,15 @@ def compute(
     )
 
     # TODO if nq > 5:
-    mapn_tracer.compute(
-        pe1,
-        pe2,
-        dp2,
-        tracers,
-        nq,
-        0.0,
+    MapN_Tracer = utils.cached_stencil_class(MapNTracer)(
```

Comment:
Should this begin with a lowercase `m`, since it is a variable name?

Reply:
yes, thanks!